applications/launchpad/backend/src/docker/models.rs

```diff
@@ -230,7 +230,9 @@ impl ImageType {
             Self::Sha3Miner => "SHA3 miner",
             Self::MmProxy => "MM proxy",
             Self::Monerod => "Monerod",
-            Self::Frontail => "Frontail",
+            Self::Loki => "loki",
```

Comment:
```suggestion
            Self::Loki => "Loki",
```

Reply:
Anyway, these changes should not reflect in this PR. 
Rather update / rebase against latest HEAD.